frontend/src/lib/router.tsx

```diff
@@ -432,7 +433,8 @@ export interface RouteURLProps {
 }
 
 export function createRouteURL(routeName: string, params: RouteURLProps = {}) {
-  const route = getRoute(routeName);
+  const storeRoutes = store.getState().ui.routes;
+  const route = getRoute(routeName) || (storeRoutes && storeRoutes[routeName]);
```

Comment:
I wonder why getRoute(routeName) is chosen over the store routes here?

Reply:
So it's like first search in the list of routes we have if that's undefined check if it's in storeRoutes